share/functions/__fish_is_git_repository.fish

```diff
@@ -0,0 +1,3 @@
+function __fish_is_git_repository --discription 'Check if current directory is a git repository'
```

Comment:
That's strange... it works with me.

Have you tried to do a `source path/to/__fish_is_git_repository.fish`? Did you put it in `~/.config/fish/functions` with the typo fixed to test it?